holidays/countries/cabo_verde.py

```diff
@@ -0,0 +1,276 @@
+
+        # Father's Day.
+        self._add_holiday_3rd_sun_of_jun(tr("Dia dos Pais"))
+
+    def _populate_subdiv_br_public_holidays(self):
+        # Law # 93/82 of November 6th
+
+        # Brava Municipality Day.
+        self._add_holiday_jun_24(tr("Dia do Município da Brava"))
+
+    def _populate_subdiv_bv_public_holidays(self):
+        # Law # 93/82 of November 6th
+
+        # Boa Vista Municipality Day.
+        self._add_holiday_jul_4(tr("Dia do Município da Boa Vista"))
+
+    def _populate_subdiv_ca_public_holidays(self):
+        # Santa Catarina de Santiago Municipality Day.
+        self._add_holiday_nov_25(tr("Dia do Município de Santa Catarina de Santiago"))
+
+    def _populate_subdiv_cf_public_holidays(self):
+        # Law # 66/VI/2005 of May 9th.
+        if self._year >= 2005:
+            # Santa Catarina do Fogo Municipality Day.
+            self._add_holiday_nov_25(tr("Dia do Município de Santa Catarina do Fogo"))
+
+    def _populate_subdiv_cr_public_holidays(self):
+        # Santa Cruz Municipality Day.
+        self._add_holiday_jul_25(tr("Dia do Município de Santa Cruz"))
+
+    def _populate_subdiv_ma_public_holidays(self):
+        # Law # 93/82 of November 6th
+
+        # Maio Municipality Day.
+        self._add_holiday_sep_8(tr("Dia do Município do Maio"))
+
+    def _populate_subdiv_mo_public_holidays(self):
+        # Law # 23/IV/91 of December 30th
+        if self._year >= 1992:
+            # Mosteiros Municipality Day.
+            self._add_holiday_aug_15(tr("Dia do Município dos Mosteiros"))
+
+    def _populate_subdiv_pa_public_holidays(self):
+        # Law # 93/82 of November 6th
+
+        # Santo Antão Island Day.
+        self._add_holiday_jan_17(tr("Dia da Ilha de Santo Antão"))
+
+        # Paúl Municipality Day.
+        self._add_holiday_jun_13(tr("Dia do Município do Paúl"))
+
+    def _populate_subdiv_pn_public_holidays(self):
+        # Law # 93/82 of November 6th
+
+        # Santo Antão Island Day.
+        self._add_holiday_jan_17(tr("Dia da Ilha de Santo Antão"))
+
+        # Porto Novo Municipality Day.
+        self._add_holiday_sep_2(tr("Dia do Município do Porto Novo"))
+
+    def _populate_subdiv_pr_public_holidays(self):
+        # Praia Municipality Day.
+        self._add_holiday_may_19(tr("Dia do Município da Praia"))
+
+    def _populate_subdiv_rb_public_holidays(self):
+        # Law # 67/VI/2005 of May 9th
+
+        # Ribeira Brava Municipality Day.
+        self._add_holiday_dec_6(tr("Dia do Município de Ribeira Brava"))
+
+    def _populate_subdiv_rg_public_holidays(self):
+        # Santo Antão Island Day.
+        self._add_holiday_jan_17(tr("Dia da Ilha de Santo Antão"))
+
+        # Ribeira Grande Municipality Day.
+        self._add_holiday_may_7(tr("Dia do Município de Ribeira Grande"))
+
+    def _populate_subdiv_rs_public_holidays(self):
+        # Law # 63/VI/2005 of May 9th
+        if self._year >= 2005:
```

Comment:
```suggestion
        if self._year >= 2006:
```
Law from May 9, holiday is on Jan 31.